src/App.jsx

```diff
@@ -1,14 +1,19 @@
 import Footer from './components/Footer/Footer';
 import Header from './components/Header/Header';
 import CurrentCityForecastView from './views/CurrentCityForecastView/CurrentCityForecastView';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 function App() {
   return (
-    <>
+    <BrowserRouter>
```

Comment:
Correct me if I'm wrong - weren't you suppose to pass history object here ?

Reply:
Documentation tell us that BrowserRouter has just five props:
- basename
- getUserConfirmation
- forceRefresh
- keyLength
- children